TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
@schellap :+1: 

If we can avoid a version downgrade that will make things simpler, especially if we need to take this change (or further servicing fixes) upstack.